style.css

```diff
@@ -184,4 +186,237 @@ header button{
         margin-top: 30%;
         margin-left: 0;
     }
- }
\ No newline at end of file
+ }
+
+
+ /*Guessing game*/
+ 
+html, .heroPicture {
```

Comment:
Here was `html, body` but I believe that was giving conflicts with the other pages. For this reason that I have done another css file and didn't connect the page that I did to the main css file. But it's ok, I believe that we just need to delete the `.heroPicture` from this line.